src/components/cart-product-card/Cart-Product-Card.js

```diff
@@ -0,0 +1,78 @@
+import {
+  axiosAddQty,
+  axiosDecrementQty,
+  axiosRemoveFromCart
+} from "../../utilty/cart-utility";
+import { axiosAddToWishlist } from "../../utilty/wishlist-utility";
+
+import { useCart } from "../../contexts/Cart-Context";
+import { useWishlist } from "../../contexts/Wishlist-Context";
+
+export function CartProductCard({ prd }) {
+  const { cartState, cartDispatch } = useCart();
+  const { wishlistItems, setwishlistItems } = useWishlist();
+
+  function getProdTotal(prod) {
+    const total = parseInt(prod.discountedPrice) * parseInt(prod.qty);
+    return total;
+  }
+
+  return (
+    <div key={prd._id} className="horizCardParent">
+      <div className="horizCardBody">
+        <div className="HorizImgDiv">
+          <img src={prd.image} />
+        </div>
+        <div className="HorizCardDetails">
+          <div className="brandTitle">{prd.name}</div>
+          <p className="prdDescrip">{prd.brand}</p>

```

Comment:
Smart way to truncate the long description text in the UI 👍

Reply:
Thank you